mix.exs

```diff
@@ -28,6 +28,7 @@ defmodule MarathonEventExporter.Mixfile do
   defp deps do
     [
       {:httpoison, "~> 0.13"},
+      {:cowboy, "~> 2.1"},
       {:excoveralls, "~> 0.7", only: :test},
```

Comment:
The `only: :test` specifier at the end. I left `cowboy` in as a full dependency because we're eventually going to need an HTTP server to export the prometheus metrics.